src/main/java/com/revature/model/Analysis_Item.java

```diff
@@ -0,0 +1,127 @@
+package com.revature.model;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "analysis_items")
+public class Analysis_Item {
```

Comment:
Please use the correct Naming scheme. Discussed yesterday with @ConnorMRyan

Reply:
I think there up to date now

Reply:
Also I added a junit test if you could look at that please